aws/sdk-codegen/src/test/kotlin/SdkCodegenIntegrationTest.kt

```diff
@@ -43,6 +43,30 @@ class SdkCodegenIntegrationTest {
             operation SomeOperation {
                 output: SomeOutput
             }
+
+            // re-use get thing output in a list & in an operation
+            @http(uri: "/SomeOperation2", method: "GET")
```

Comment:
Organizationally speaking, this test belongs in codegen-client rather than sdk-codegen. Looks like we also need an equivalent test for inputs.

Reply:
oh right good point—when I started debugging I thought the SDK was involved